libs/knowledge-store/tests/integration_tests/test_graph_store.py

```diff
@@ -21,14 +21,30 @@ def cassandra() -> Iterator[LocalCassandraTestStore]:
         store.docker_container.stop()
 
 
+class DummyEmbeddingModel(EmbeddingModel):
+    def embed_texts(self, _: List[str]) -> List[List[float]]:
+        return []
```

Comment:
This won't work -- returning empty embeddings is invalid. In a PR I'm working on I added a dependency on `ragstack-langchain` so this could use the adapter there.

Reply:
I don't think it's a good thing to have a dependency on ragstack-langchain. That's a circular dependency...
For the test, we only need embed_query to return a non-zero length vector so I return a fixed one.

Reply:
OK. We can use OpenAI or HuggingFace later when we need it. Keeping the test dependency on ragstack-ai-langchain for now.

Reply:
We'll probably remove the graph store from ragstack-ai-langchain at some point.
So for tests I think it would probably be better to use langchain directly.